networkdb/networkdb_test.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/sirupsen/logrus"
 	"gotest.tools/assert"
 	is "gotest.tools/assert/cmp"
+	"k8s.io/apimachinery/pkg/util/wait"
```

Comment:
instead of this, could you use the gotest.tools utilities? https://godoc.org/gotest.tools/poll#WaitOn

Reply:
ya thought about it, saw some of the wrappers in `moby` as well, found this API to be simpler

Reply:
I don't think we should mix/match testing frameworks, and adding more dependencies if not needed

Reply:
ok:(